webapp/src/app/pages/landing-page/landing-page.component.ts

```
@@ -84,19 +84,31 @@
             (user) => user.role.name === 'moderator'
           );
-  console.log(' userModerator: ', userModerator);
+      if (!userModerator) {
+        throw new Error(`Moderator not found for course with id ${course.id}`);
+      }
           this.buildCardInfo(userModerator.userId, course);
         }
+       catch (error) {
+        
+        console.log('Error:', error);  
+      continue
```

Biome (1.9.4)

[error] Unnecessary continue statement

Unsafe fix: Delete the unnecessary continue statement


(lint/correctness/noUnnecessaryContinue)